### PR TITLE
Refactor assets to more appropriate folders

### PR DESCRIPTION
**Pull request by MattyTheHacker**
moves the sponsorship images to the new sponsorship assets folder along with the sponsorship proposal
moves the constitution out of the root and into the assets folder

### CODE REVIEW

File: _data/redirects.yaml [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 # sponsor
 - source: /sponsor
-  destination: /CSS_Sponsorship_Proposal.pdf
```
Comment (LMBishop) [inline _data/redirects.yaml:7]: Maybe add a redirect for this as well

Comment (MattyTheHacker) [reply _data/redirects.yaml:7]: You mean if someone tries to navigate directly to cssbham.com/CSS_Sponsorship_Proposal.pdf ?

Comment (LMBishop) [reply _data/redirects.yaml:7]: yes

Comment (MattyTheHacker) [reply _data/redirects.yaml:7]: yeah good point tbf, will do

**Comment by MattyTheHacker** [comment]
Hmm, that doesn't seem to have worked

**Comment by LMBishop** [comment]
Maybe you can't redirect to a redirect

**Comment by MattyTheHacker** [comment]
Mm idk I thought cos the redirect happens at the java script level that it'd be fine but I'm not sure. I'll have another look in the morning

**Comment by MattyTheHacker** [comment]
Can't seem to figure out how to get it to work when the source of the re-direct is a file so have just removed it

**Review by LMBishop — APPROVED** [review]

**Pull request merged**